tests/testsite/testsite/settings/django.py

```diff
@@ -67,6 +67,8 @@
     }
 }
 
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
Is this new? Looks unrelated to the test changes?

Reply:
This is new; I believe this setting is required in Django 4.

Reply:
Have rebased & dropped the Django 4 stuff; this disappears for now.